api/v1alpha1/registeredservice_types.go

```diff
@@ -25,26 +25,67 @@ type RegisteredServiceConstraints struct {
     Environments []string `json:"environments,omitempty"`
 }
 
+type HealthCheckContainer struct {
+    // Container image with the client to run the test
+    Image string `json:"image"`
+    // Command to execute in the container to run the test
+    Cmd string `json:"cmd"`
```

Comment:
nit: The struct member name _should_ match the documentation, like so:

```suggestion
    // Command to execute in the container to run the test
    Command string `json:"cmd"`
```

Reply:
OK